src/script/api/script_tile.hpp

```diff
@@ -351,7 +351,7 @@ class ScriptTile : public ScriptObject {
 	 * @pre width > 0.
 	 * @pre height > 0.
 	 * @pre radius >= 0.
-	 * @return Value below 8 means no acceptance; the more the better.
+	 * @return Value below 8 mean no acceptance; the more the better.
```

Comment:
```suggestion
	 * @return Values below 8 mean no acceptance; the more the better.
```
Or perhaps no change at all